mir_eval/separation.py

```diff
@@ -442,6 +443,102 @@ def bss_eval_images(reference_sources, estimated_sources,
         return (sdr, isr, sir, sar, popt)
 
 
+def bss_eval_images_framewise(reference_sources, estimated_sources,
+                              window=30*44100, hop=15*44100,
+                              compute_permutation=False):
+    """Framewise computation of bss_eval_images
+
```

Comment:
It would make most sense to put that in the `bss_eval_images` docstring. Are we sure that link will be valid for a long time? Maybe mention that link and the paper in case the link goes belly up?